src/test/java/racingGame/CarsTest.java

```diff
@@ -0,0 +1,74 @@
+package racingGame;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CarsTest {
+
+    @Test
+    @DisplayName("fromNames로 생성하면 각 이름에 해당하는 Car들을 가진다")
+    void fromNames_createsCars() {
+        List<String> names = Arrays.asList("a", "b", "c");
+
+        Cars cars = Cars.fromNames(names);
+
+        assertThat(cars.asList())
+            .hasSize(3)
+            .extracting(Car::name)
+            .containsExactly("a", "b", "c");
+    }
+
+    @Test
+    @DisplayName("moveAll을 호출하면 모든 Car에 전략이 적용되어 이동한다")
+    void moveAll_movesAllCars() {
+        List<String> names = Arrays.asList("a", "b");
+        Cars cars = Cars.fromNames(names);
+
+        cars.moveAll(() -> true);
+
+        assertThat(cars.asList())
+            .extracting(Car::position)
+            .containsExactly(1, 1);
+    }
+
+    @Test
+    @DisplayName("winnerNames는 최대 위치를 가진 Car의 이름들만 반환한다")
+    void winnerNames_returnsNamesWithMaxPosition() {
+        List<String> names = Arrays.asList("a", "b", "c");
+        Cars cars = Cars.fromNames(names);
+
+        // a : 2칸, b : 1칸, c : 2칸
+        cars.asList().get(0).move(() -> true);
+        cars.asList().get(0).move(() -> true);
+
+        cars.asList().get(1).move(() -> true);
+
+        cars.asList().get(2).move(() -> true);
+        cars.asList().get(2).move(() -> true);
+
+        List<String> winners = cars.winnerNames();
```

Comment:
```suggestion
        Car pobi = new Car("pobi", 5);
        Car jason = new Car("jason", 4);
        Car woni = new Car("woni", 3);
        Cars cars = new Cars(List.of(pobi, jason, woni));

        Winners winners = cars.findWinners();
```
수업 중에 다룬 제 코드 피드백해 봄
경우가 끝났을 때의 상태 값을 위와 같이 초기화할 수 있다면 우승자를 구한기 위한 Test Fixture를 정훈님이 구현한 코드와 비교해 더 쉽게 생성한 후 테스트 가능하지 않을까?